nautobot_golden_config/models.py

```diff
@@ -795,7 +786,7 @@ def __str__(self):
 @extras_features(
     "graphql",
 )
-class RemediationSetting(PrimaryModel):  # pylint: disable=too-many-ancestors
+class RemediationSetting(OrganizationalModel):  # pylint: disable=too-many-ancestors
```

Comment:
??

Reply:
based on https://github.com/nautobot/nautobot-plugin-golden-config/pull/533/files#r1316280651

Reply:
I think this is just a miss, we should have tags on this.

Reply:
cc @whitej6 

similar discussion  : https://github.com/nautobot/nautobot-plugin-bgp-models/pull/109#discussion_r1311612077

Reply:
👍 for tags on this one , would like to understand why we would not allow them ?

Reply:
Yes, the alternative to my comment to switch this to OrganizationalModel is to actually add a tag field to all of the forms. I did not initially review this portion of the plugin, so I assumed tags were intentionally left off of this model and no one noticed the empty tag section on the detail view with no way of changing it.